src/main/java/ro/societateahermes/backendservice/utils/mapper/OfferMapper.java

```diff
@@ -0,0 +1,40 @@
+package ro.societateahermes.backendservice.utils.mapper;
+
+import org.springframework.stereotype.Component;
+import ro.societateahermes.backendservice.entities.Offer;
+import ro.societateahermes.backendservice.entities.dto.OfferDTO;
+import ro.societateahermes.backendservice.repository.EventRepositoryInterface;
+
+@Component
+public class OfferMapper {
+
+    private SponsorMapper sponsorMapper;
+    private EventRepositoryInterface eventRepo;
```

Comment:
Cine v-a dat ma ideea asta creata ca ar trebui sa aveti un repo in mapper? :)) Daca nu il folositi scoateti-l din clasa

Reply:
Aveam nevoie de el pentru sponsormapper:)))

Reply:
of bine :)) Am clarificat si cu gabi care-i treaba si e ok